<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>645</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>V&amp;V Framework</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>127</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="EvaEnvPB">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Evaluation
Environment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EvaTypePB">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Evaluation
Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="TypeCompPB">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Type of Component
under
Evaluation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EvaToolPB">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Evaluation
Tool</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EvaStagePB">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Evaluation
Stage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LogicPB">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Logic of the
Component</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1. Dimension&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;2. Dimension&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;3. Dimension&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;4. Dimension&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;5. Dimension&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;6. Dimension&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeWidget" name="EvaEnvTW">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Layers</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>In-the-Lab</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Closed</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Open</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="EvaTypeTW">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Layers</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Experimental</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Analytical</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="TypeCompTW">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Layers</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Software</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hardware</string>
     </property>
     <item>
      <property name="text">
       <string>Sensor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MCU</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Camera</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Model</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="EvaToolTW">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Layers</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Open source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Proprietary</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="EvaStageTW">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Layers</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Verification</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Validation</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="LogicTW">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Layers</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Sensing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thinking</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Acting</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>111</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Layers Selected</string>
    </property>
    <widget class="QLabel" name="Dim1SelectedLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>90</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
border-color: rgb(114, 159, 207);
color: rgb(78, 154, 6);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>1. Dimension</string>
     </property>
    </widget>
    <widget class="QLabel" name="Dim2SelectedLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>90</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
border-color: rgb(114, 159, 207);
color: rgb(78, 154, 6);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>2. Dimension</string>
     </property>
    </widget>
    <widget class="QLabel" name="Dim3SelectedLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>90</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
border-color: rgb(114, 159, 207);
color: rgb(78, 154, 6);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>3. Dimension</string>
     </property>
    </widget>
    <widget class="QLabel" name="Dim4SelectedLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>90</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
border-color: rgb(114, 159, 207);
color: rgb(78, 154, 6);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>4. Dimension</string>
     </property>
    </widget>
    <widget class="QLabel" name="Dim5SelectedLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>90</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
border-color: rgb(114, 159, 207);
color: rgb(78, 154, 6);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>5. Dimension</string>
     </property>
    </widget>
    <widget class="QLabel" name="Dim6SelectedLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>90</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
border-color: rgb(114, 159, 207);
color: rgb(78, 154, 6);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>6. Dimension</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="GenerateXMLPB">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>125</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>Generate XML</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="FileNameTE">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>560</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Untitled.xml</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
